python/ray/autoscaler/_private/util.py

```diff
@@ -362,17 +362,18 @@ def format_info_string(lm_summary, autoscaler_summary, time=None):
     for node_type, count in autoscaler_summary.pending_launches.items():
         line = f" {node_type}, {count} launching"
         pending_lines.append(line)
-    for ip, node_type, status in autoscaler_summary.pending_nodes:
-        line = f" {ip}: {node_type}, {status.lower()}"
+    for ip, node_type, status, log_path in autoscaler_summary.pending_nodes:
+        line = f" {ip}: {node_type}, {status.lower()}, {log_path}"
         pending_lines.append(line)
     if pending_lines:
         pending_report = "\n".join(pending_lines)
     else:
         pending_report = " (no pending nodes)"
 
     failure_lines = []
-    for ip, node_type in autoscaler_summary.failed_nodes:
-        line = f" {ip}: {node_type}"
+    for ip, node_type, log_path in autoscaler_summary.failed_nodes:
+        line = f" {ip}: {node_type}, {log_path}"
+        failure_lines.append(line)
```

Comment:
awkward latent bug

Reply:
The user will basically go to this log path in /tmp/ray/session_latest/logs/<worker-id>.log to see what errored?

Reply:
yeah i think the suffix is `.out` instead of `.log` but yeah

Reply:
why not .log for consistency?

Reply:
as mentioned below, this is more consistent